CMakeLists.txt

```diff
@@ -181,7 +180,14 @@ endif()
 #     products/zerocal.cpp
 #     products/gravitycal.cpp
 #     ... others?
-set(config_dir /etc/barrett/)
+set(BARRETT_ETC_PATH /etc/barrett)
+configure_file(
+  ${PROJECT_SOURCE_DIR}/include/barrett/config.h.in 
+  ${PROJECT_BINARY_DIR}/include/barrett/config.h)
+install(FILES ${PROJECT_BINARY_DIR}/include/barrett/config.h 
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/include/barrett)
+
+set(config_dir ${BARRETT_ETC_PATH})
```

Comment:
In our previous meeting, we have decided that it is up to the user to populate and maintain the ~/.barrett directory and make install installs the config files to /etc/barrett/